examples/helm/build-all.sh

```diff
@@ -8,7 +8,9 @@ function build_charts () {
   list_dirs | while read CHART; do
     if [ -f ${CHART}/Makefile ]; then make -C ${CHART}; fi
     if [ -f ${CHART}/requirements.yaml ]; then helm dep up ${CHART}; fi
-    helm package ./${CHART}
+    if [ -f ${CHART}/Chart.yaml ]; then 
+      helm package ./${CHART};
```

Comment:
Initially I've put it inline, like the ones above. Afterwards made it more visible, for people to understand this is the main purpose of running this build as well as a "provision" for future addition like else echo "no chart has been found under folder $CHART.